### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by wtritch**
This PR includes some minor cleanup that were pointed out in previous PDF batch generation reviews.

### CODE REVIEW

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/report/BatchExamReportRequestHolder.java [hunk 1]
```diff
@@ -10,6 +11,8 @@
     private PermissionScope permissionScope;
     private String user;
     private String label;
+
+    @JsonTypeInfo(use=JsonTypeInfo.Id.CLASS, include=JsonTypeInfo.As.PROPERTY, property="@class")
```
Comment (wtritch) [inline common/src/main/java/org/opentestsystem/rdw/reporting/common/report/BatchExamReportRequestHolder.java]: @esotrope I moved the typing off of the AbstractExamReportRequest class and onto this BatchExamReportRequestHolder property, which should remove the need for the workaround in your existing PR.

File: pdf-report-processor/src/main/java/org/opentestsystem/rdw/reporting/processor/repository/JdbcReportRepository.java [hunk 2]
```diff
@@ -54,7 +55,8 @@
     public JdbcReportRepository(final NamedParameterJdbcTemplate template,
                                 final ObjectMapper objectMapper) {
         this.template = template;
-        this.objectMapper = objectMapper;
+        this.objectMapper = objectMapper.copy();
+        this.objectMapper.enableDefaultTypingAsProperty(NON_FINAL, "@class");
```
Comment (wtritch) [inline pdf-report-processor/src/main/java/org/opentestsystem/rdw/reporting/processor/repository/JdbcReportRepository.java]: Now that we aren't including type information on all AbstractExamReportRequest instances by default, we need to manually include type information when persisting the serialized report request as a Report column value.

File: pdf-report-processor/src/main/resources/application.yml [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 # since those are the only end-points, switch to a less standard port using the main server.port config
 management:
-  port: 8009
+  port: 8008
   security:
     enabled: false
 
 server:
-  port: 8083
+  port: 8080
```
Comment (wtritch) [inline pdf-report-processor/src/main/resources/application.yml]: @malaffoon reverted this to standard application ports.

**Comment by wtritch** [comment]
Merging this for time, will continue to resolve comments as they come in.

File: pdf-report-processor/src/main/resources/application.yml (hunk 3 above)
Comment (malaffoon) [inline pdf-report-processor/src/main/resources/application.yml]: 👍

**Pull request merged**